lib/install.js

```diff
@@ -382,11 +383,13 @@ Installer.prototype.loadAllDepsIntoIdealTree = function (cb) {
     steps.push([validateArgs, this.idealTree, this.args])
     steps.push([loadRequestedDeps, this.args, this.idealTree, saveDeps, cg.newGroup('loadRequestedDeps')])
   } else {
-    if (this.prod) {
+    if (this.prod && this.dev) {
```

Comment:
Yes, your example would produce the same result. The difference is that it would _always_ produce that result no matter what options you passed into npm.

I think having it not always select the most recent version it _could_ use is ok, I just don't want different results for dev and prod environments.